src/write_grpc.c

```diff
@@ -397,42 +398,29 @@ static gpr_timespec get_deadline(cdtime_t delay)
  * accepted or cq is empty.  Returns a true value if all writes were
  * accepted.  Must call with cb->mutex held. */
 static int process_cq(grpc_callback *cb, int do_sleep) {
-  grpc_event *ev;
+  grpc_event ev;
 
   while (cb->write_accepted_counter < cb->write_counter) {
```

Comment:
Removed them, and revamped the completion-queue wait and drain logic. PTAL at that, too (process_cq, drain_cq, and calls to them).